README.md

```diff
@@ -1,4 +1,6 @@
-# The Nu Html Checker (v.Nu) [![Chat room][1]][2] [![Download latest][3]][4]
+# The Nu Html Checker (v.Nu)
+
+[![Linux Build Status](https://img.shields.io/travis/validator/validator/master.svg?label=Linux%20build)](https://travis-ci.org/validator/validator) [![Windows Build status](https://img.shields.io/appveyor/ci/sideshowbarker/validator/master.svg?label=Windows%20build)](https://ci.appveyor.com/project/sideshowbarker/validator) [![Chat room][1]][2] [![Download latest][3]][4]
 
```

Comment:
@XhmikosR  maybe add docker badge as well?

Reply:
The final decision is @sideshowbarker's. I definitely want to have the CI badges so that I can find the CI projects easily.

Reply:
Sure. I'ts a similar thing with the docker hub image, which is linked further down in the readme, but there's no quick way to find it atm. I'd be happy to add it if @sideshowbarker is fine with it.